Formula/autoenv.rb

```diff
@@ -16,4 +16,11 @@ def caveats; <<-EOS.undent
       source #{opt_prefix}/activate.sh
     EOS
   end
+
+  test do
+    (testpath/"test").mkpath
+    (testpath/"test/.env").write "echo it works\n"
+    testcmd = "yes | bash -c '. #{opt_prefix}/activate.sh; autoenv_cd test'"
```

Comment:
if it's for the formula being tested, not some other formula, I prefer `prefix` over `opt_prefix`